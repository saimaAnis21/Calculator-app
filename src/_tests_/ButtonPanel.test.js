import React from 'react';
import renderer from 'react-test-renderer';
import ButtonPanel from '../components/ButtonPanel';

describe('ButtonPanel renders correctly', () => {
  const test = () => { };

  it('renders correctly', () => {
    const tree = renderer
      .create(<ButtonPanel clickHandler={test} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
