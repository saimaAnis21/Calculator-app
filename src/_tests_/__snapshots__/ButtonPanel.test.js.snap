// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonPanel renders correctly renders correctly 1`] = `
<div
  className="w-100p d-flex flex-col"
>
  <div
    className="h-100"
  >
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      AC
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      +/-
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      %
    </button>
    <button
      className="orng-btns w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      /
    </button>
  </div>
  <div
    className="h-100"
  >
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      7
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      8
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      9
    </button>
    <button
      className="orng-btns w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      X
    </button>
  </div>
  <div
    className="h-100"
  >
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      4
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      5
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      6
    </button>
    <button
      className="orng-btns w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      -
    </button>
  </div>
  <div
    className="h-100"
  >
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      1
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      2
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      3
    </button>
    <button
      className="orng-btns w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      +
    </button>
  </div>
  <div
    className="h-100"
  >
    <button
      className="bg-btnpanel w-50p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      0
    </button>
    <button
      className="bg-btnpanel w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      .
    </button>
    <button
      className="orng-btns w-25p h-100p brdr"
      onClick={[Function]}
      type="submit"
    >
      =
    </button>
  </div>
</div>
`;
